CHANGELOG.md

```diff
@@ -6,12 +6,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unpublished
 
-## 0.1.2 - 10-9-2020
+## 0.1.2-alpha.0 - 10-27-2020
```

Comment:
Should be 0.1.3-alpha.0 - keep the 0.1.2 history (unless I'm reading this diff wrong)